modules/mixed-arch-import-imagestreams.adoc

```diff
@@ -0,0 +1,74 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="mixed-arch-import-imagestreams_{context}"]
+
+= Importing manifest lists in image streams on your heterogeneous cluster 
+
+On a {product-title} {product-version} heterogeneous cluster, the image streams in the cluster do not import manifest lists automatically. You must manually change the default `importMode` option to the `PreserveOriginal` option in order to import the manifest list.
+
+[NOTE]
+====
+For this procedure to run successfully, your `referencePolicy` object must be set to the `Source` type in your `imagestreams.yaml` file:
```

Comment:
Maybe better to reuse object and field as in the following?
``` the referencePolicy.type field of your ImageStream object must be set to Source```

Reply:
yeah definitely sounds better and more understandable from an API perspective.